.github/workflows/lighthouse-ci-workflow.yml

```diff
@@ -1,19 +1,9 @@
 name: Lighthouse CI
 
-# By default, runs when a pull_request's activity type is opened, synchronize,
-# or reopened and on main so comparisons to ancestors can be made.
 on:
   push:
     branches:
```

Comment:
This is actually interesting; what value does this action have at all right now? Like, I feel like doing this _again_ for every PR after it's merged is just burning CPU.

Reply:
I did agree except for the fact "on main so comparisons to ancestors can be made."

I'm assuming we want a log/history of commits to main.

Reply:
Yes, but... we don't really do this right now. Like, do we even get an email if this fails after merge?